apps/cbportal/base/CBGrid/Virtual Machines/VMConsole.wiki

```diff
@@ -0,0 +1,21 @@
+@usedefaults
+{{vmachine}}
+h2. Virtual Machine: ${name}
+{% if status != "DESTROYED" %}
```

Comment:
Should probably check for deleted as well

Reply:
This is required for the restore action